File: Backend/Models/Telefon.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Backend.Models{
    public class Telefon{

        [Key]
        [Column("ID")]
        public int id { get; set; }
        
        [Column("Broj")]
        [MaxLength(15)]
        public string broj { get; set; }

        [Column("Tip")]
        [MaxLength(30)]
        public string tip { get; set; }

        [JsonIgnore]
        [Column("Kontakt")]
        public Kontakt kontakt {get; set; }
        
        
        
    }
}

File: Frontend/telefon.js
export class Telefon{
    
    constructor(broj, tip){
        this.id = -1;
        this.broj = broj;
        this.tip = tip;
    }

        izmeniTelefon(broj, tip){
            this.broj = broj;
            this.tip = tip;
        }

        istiBrojTip(broj,tip){
        if(this.broj === broj && this.tip === tip)
            return true;
        else
            return false;
    }
}